misk/src/main/kotlin/misk/web/actions/LivenessCheckAction.kt

```diff
@@ -10,19 +10,18 @@ import javax.inject.Inject
 import javax.inject.Singleton
 
 @Singleton
-class LivenessCheckAction : WebAction {
-  @Inject private lateinit var services: MutableSet<Service>
+class LivenessCheckAction @Inject internal constructor(
+  private val services: List<Service>
+) : WebAction {
 
   @Get("/_liveness")
   @ResponseContentType(MediaTypes.TEXT_PLAIN_UTF8)
   fun livenessCheck(): Response<String> {
     val isAlive = services.all {
       it.state() != State.FAILED && it.state() != State.TERMINATED
     }
+
     // TODO(jgulbronson) - Should return an empty body
```

Comment:
TODO done?

Reply:
Alternately, we could just make a ResponseBody.EMPTY constant that has no content-type and is 0 bytes long